<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceControl</class>
 <widget class="QMainWindow" name="ServiceControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServiceControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="paneSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="domainsGroupBox">
       <property name="title">
        <string>Domains</string>
       </property>
       <layout class="QVBoxLayout" name="domainsPane">
        <item>
         <widget class="QListView" name="domainList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="domainActions">
          <item>
           <widget class="QPushButton" name="addDomainButton">
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeDomainAction">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="domainActionsSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="unitListGroupBox">
       <property name="title">
        <string>Units</string>
       </property>
       <layout class="QVBoxLayout" name="unitsPane">
        <item>
         <widget class="QListView" name="unitList"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="unitGroupBox">
       <property name="title">
        <string>Unit</string>
       </property>
       <layout class="QVBoxLayout" name="unitPane">
        <item>
         <widget class="QGroupBox" name="unitStatusGroupBox">
          <property name="title">
           <string>Status</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTextBrowser" name="unitStatusView">
             <property name="styleSheet">
              <string notr="true">QTextBrowser {
    background-color: transparent;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot; bgcolor=&quot;transparent&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="unitActionsGroupBox">
          <property name="title">
           <string>Actions</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="startUnitButton">
             <property name="text">
              <string>&amp;Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopUnitButton">
             <property name="text">
              <string>S&amp;top</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="restartUnitButton">
             <property name="text">
              <string>&amp;Restart</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="reloadUnitButton">
             <property name="text">
              <string>Re&amp;load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="killUnitButton">
             <property name="text">
              <string>&amp;Kill...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="unitLogsGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Logs</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTextBrowser" name="unitLogsView">
             <property name="styleSheet">
              <string notr="true">QTextBrowser {
    background-color: transparent;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServiceControl">
    <property name="title">
     <string>&amp;Service Control</string>
    </property>
    <addaction name="actionAddDomain"/>
    <addaction name="actionRemoveDomain"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Unit">
    <property name="title">
     <string>&amp;Unit</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionRestart"/>
    <addaction name="actionReload"/>
    <addaction name="actionKill"/>
   </widget>
   <addaction name="menuServiceControl"/>
   <addaction name="menu_Unit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddDomain">
   <property name="text">
    <string>&amp;Add Domain...</string>
   </property>
  </action>
  <action name="actionRemoveDomain">
   <property name="text">
    <string>&amp;Remove Domain</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>S&amp;top</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>&amp;Restart</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Re&amp;load</string>
   </property>
  </action>
  <action name="actionKill">
   <property name="text">
    <string>&amp;Kill...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
